go.mod

```diff
@@ -1,3 +1,5 @@
 module github.com/norunners/vert
 
 go 1.16
+
+require github.com/corbym/gocrest v1.0.5 // indirect
```

Comment:
Remove this test dependency.

Reply:
It's not possible to remove this one - it would break the provided test below.
I guess we're talking about introducing a new dependency here. Please, let me re-locate this conversation into the "main" PR dialog.

Reply:
This library is simple enough to have zero dependencies. I'll look into updating your tests if you like.

Reply:
Hi, thank you for answering.

Well, from engineer to engineer, I can't agree on "This library is simple enough to have zero dependencies."
Since this is a "test only" dependency and test-support libraries like this have the tendency to be stable over a long period of time. The major benefit I see is improved expressiveness when reading tests.
Hence, I consider the value of following the clean code principle is the stronger goal,
in relation to zero dependencies (as in all dependencies).
(just my 2cents)

That said, this is your lib, and you have the last word about the goals and the non-goals of this lib.

If you wish, I can rework towards "vanilla go" and remove the dependency as well, or you do this ... as you wish.

Happy to chat further, if you want :)